lib/zlib.js

```diff
@@ -326,15 +332,8 @@ function Zlib(opts, mode) {
     }
   }
 
-  if (opts.strategy) {
-    if (opts.strategy != constants.Z_FILTERED &&
-        opts.strategy != constants.Z_HUFFMAN_ONLY &&
-        opts.strategy != constants.Z_RLE &&
-        opts.strategy != constants.Z_FIXED &&
-        opts.strategy != constants.Z_DEFAULT_STRATEGY) {
-      throw new Error('Invalid strategy: ' + opts.strategy);
-    }
-  }
+  if (opts.strategy && !(strategies.includes(opts.strategy)))
```

Comment:
A range check may be faster than `includes` ...e.g. 

```
if (opts.strategy >= constants.Z_DEFAULT_STRATEGY &&
   opt.strategy <= constants.Z_FIXED) { ... }
```
A bit more work to maintain if these values ever change but it avoids the linear scan of the array that `includes()` requires.